src/sentry/runner/commands/backup.py

```diff
@@ -51,7 +53,9 @@ def sort_dependencies():
     """
     from django.apps import apps
 
-    from sentry.models import Actor, Team, User
```

Comment:
you like this better? 😛

Reply:
flake8 was complaining about it (S005 I think), so I just fixed it in the way it specified